src/rest/v0/block/mod.rs

```diff
@@ -16,10 +15,12 @@ pub fn create_handler(
     blockchain: BlockchainR<Mockchain>,
 ) -> impl Fn(&str) -> App<BlockchainR<Mockchain>> + Send + Sync + Clone + 'static {
     move |prefix: &str| {
-        let path = format!("{}/v0/block/{{block_id}}", prefix);
         App::with_state(blockchain.clone())
-            .filter(PathPredicate::for_pattern(&path))
-            .resource(&path, |r| r.get().with(handle_request))
+            .prefix(prefix.to_string())
```

Comment:
The prefix should be full `format!("{}/v0/block", prefix)` like in other routes. Now this app consumes everything starting with configured prefix, for example `api/v0/node/stats`, which stopped working and now returns 404.